### PR TITLE
Expand PHPCS usage

### PR DESCRIPTION
**Pull request by AnaelMobilia**
Run PHPCS on most code of the application
Update PHPCS running configuration to use blacklist :
  - check __all__ files
  - except files which aren't under GLPI management : /files/* /lib/* /vendor/*

Fixs was made using phpcbf

### CODE REVIEW

**Comment by trasher** [comment]
Hello, thank you for this PR :)

I wonder if this is a really good idea to use phpcs to check CSS and JS files. WOuld'nt we use specific lint tools for those ones?

**Comment by AnaelMobilia** [comment]
@trasher [RT(F)M](https://github.com/squizlabs/PHP_CodeSniffer) ;-)
```
the main phpcs script that tokenizes PHP, JavaScript and CSS files to detect violations of a defined coding standard,
```

;-)

**Comment by trasher** [comment]
OK, you got the point :D

File: install/update.php [hunk 1]
```diff
@@ -570,17 +570,17 @@ function updateDbUpTo031() {
                 FROM `glpi_config`";
       $result = $DB->queryOrDie($query, "get current version");
 
-      $current_version = trim($DB->result($result,0,0));
-      $glpilanguage    = trim($DB->result($result,0,1));
-   // < 0.85
+      $current_version = trim($DB->result($result, 0, 0));
+      $glpilanguage    = trim($DB->result($result, 0, 1));
+      // < 0.85
```
Comment (trasher) [inline install/update.php]: I wonder if this one should not be in the else below.

File: install/update.php [hunk 2]
```diff
@@ -929,15 +929,15 @@ function updateTreeDropdown() {
...
+      echo "<h3><span class='red'>".sprintf(__('Caution! You will update the GLPI database named: %s'), $DB->dbdefault) ."</h3>";
 
       echo "<form action='update.php' method='post'>";
       echo "<input type='submit' class='submit' name='continuer' value=\"".__('Continue')."\">";
       Html::closeForm();
       echo "</div>";
    }
 
-// Step 2
+   // Step 2
```
Comment (trasher) [inline install/update.php]: Should be in the else

File: tools/findtableswithoutclass.php [hunk 3]
```diff
@@ -37,17 +37,16 @@
 
 include ('../inc/includes.php');
 
-#echo getSingular('criteria')."\n";
-#echo getPlural('criterias')."\n";
+// echo getSingular('criteria')."\n";
```
Comment (trasher) [inline tools/findtableswithoutclass.php]: My be removed

File: tools/generate_bigdump.function.php [hunk 4]
```diff
@@ -246,10 +246,11 @@ function addNetworkEthernetPort($itemtype, $items_id, $entities_id, $locations_i
       $nn = new Networkport_Networkport();
       $nn->add(array('networkports_id_1' => $refportID,
                      'networkports_id_2' => $newportID,));
-   } else {
-      if ($locations_id) {
-      }
    }
+   //else {
```
Comment (trasher) [inline tools/generate_bigdump.function.php]: Remove

**Comment by trasher** [comment]
Hum... I had some issues running the phpcs command on my terminal, because of the `*` character. I've removed it and added config dirs to exclude list also.

**Comment by AnaelMobilia** [comment]
@trasher Your commit doesn't remove the wildcard on previous paths ? (eg /vendor/*).

Does that configuration work on your computer (linux ?) or is it a missing ?

**Comment by trasher** [comment]
Erf, I do not know what I've done... No, wilcards must be all removed to work as expected here :) (with wilcards, vendor and plugins are checked... I don't want that :D)

**Comment by trasher** [comment]
Fixed; tanks @AnaelMobilia ;)

**Pull request merged**